### PR TITLE
Fix #1533, #1522: Glue ingestion supports Location and Glue ingestion doesn't work with AWS temporary security credentials.

### LINKED ISSUE #1533: Glue ingestion supports Location
This is a spin-off of this #1442

- [ ] gather the requirements for the glue.json tables (I have created just one with two columns)
- [ ] Ingestion of location with the Glue connector

### PR DESCRIPTION
**Pull request by amiorin**
### Describe your changes :

- Glue works with columns containing dots.
- Glue works with temporary credentials.
- Fix bug in TableRepository, the EntityType was database instead of table.
- only region_name is mandatory in Glue config.
- first integration test for Glue. The CI with AWS is still missing.

#
### Type of change :

- [x] Bug fix

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@harshach @ayush-shah @pmbrull

### CODE REVIEW

File: bootstrap/bootstrap_storage.sh [hunk 1]
```diff
@@ -35,10 +35,15 @@ else
 fi
 
 TABLE_INITIALIZER_MAIN_CLASS=org.openmetadata.catalog.util.TablesInitializer
-for file in "${BOOTSTRAP_DIR}"/../libs/*.jar;
-do
-    CLASSPATH="$CLASSPATH":"$file"
-done
+LIBS_DIR="${BOOTSTRAP_DIR}"/../libs/
```
Comment (harshach) [inline bootstrap/bootstrap_storage.sh:38]: What's the reason behind this. Lets not mix changes for one PR into another, if its needed lets open separate PR

File: ingestion/src/metadata/ingestion/source/glue.py [hunk 2]
```diff
@@ -199,14 +238,14 @@ def get_downstream_tasks(self, task_unique_id, tasks):
...
-                        ][:63]
+                        ][:64]
                     )
         return downstreamTasks
 
     def get_tasks(self, tasks):
         taskList = []
         for task in tasks["Graph"]["Nodes"]:
-            task_name = task["Name"][:63]
+            task_name = task["Name"][:64]
```
Comment (harshach) [inline ingestion/src/metadata/ingestion/source/glue.py:248]: taskname increase task.name to 128 as well in schema and here

**Comment by harshach** [comment]
LGTM except for bootstrap-storage changes. @ayush-shah review this please

File: bootstrap/bootstrap_storage.sh (hunk 1 above)
Comment (amiorin) [reply bootstrap/bootstrap_storage.sh:38]: Ok

File: ingestion/src/metadata/ingestion/source/glue.py (hunk 2 above)
Comment (amiorin) [reply ingestion/src/metadata/ingestion/source/glue.py:248]: Ok

**Review by harshach — APPROVED** [review]

**Pull request merged**